geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommand.java

```diff
@@ -433,25 +430,24 @@ private void setCacheWriter(RegionAttributesType attributes) {
   public static class Interceptor extends AbstractCliAroundInterceptor {
 
     @Override
-    public Result preExecution(GfshParseResult parseResult) {
+    public Object preExecution(GfshParseResult parseResult) {
```

Comment:
When returning Object (e.g. due to two possible return types), please include javadoc comment documenting the possible return types, and maybe some details about why this is temporary.

If it's more than temporary, I would suggest a marker interface implemented by all possible return types, rather than Object.

Reply:
it's temporary. Will change that in the next commit.